easypost/constant.py

```diff
@@ -36,6 +36,9 @@
     "FedexAccount",
     "FedexSmartpostAccount",
 ]
+_CARRIER_ACCOUNT_TYPES_WITH_CUSTOM_OAUTH = [
```

Comment:
question: do we know if other carriers will reuse this same endpoint? Otherwise this name may be too generic

Reply:
I don't know; there could be a carrier that uses this endpoint in the future. This name is similar to what we have for FedEx `_CARRIER_ACCOUNT_TYPES_WITH_CUSTOM_WORKFLOWS`